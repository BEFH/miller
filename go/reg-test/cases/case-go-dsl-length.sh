echo "x=1,y=abcdefg,z=3" | run_mlr put '$n = length($x)'
echo "x=1,y=abcdefg,z=3" | run_mlr put '$n = length($y)'
echo "x=1,y=abcdefg,z=3" | run_mlr put '$n = length($nonesuch)'
echo "x=1,y=abcdefg,z=3" | run_mlr put '$n = length($*)'
echo "x=1,y=abcdefg,z=3" | run_mlr put '$n = length([])'
echo "x=1,y=abcdefg,z=3" | run_mlr put '$n = length([5,6,7])'
echo "x=1,y=abcdefg,z=3" | run_mlr put '$n = length({})'
echo "x=1,y=abcdefg,z=3" | run_mlr put '$n = length({"a":5,"b":6,"c":7})'
