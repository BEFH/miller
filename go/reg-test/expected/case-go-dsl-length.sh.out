
mlr put $n = length($x)
x=1,y=abcdefg,z=3,n=1

mlr put $n = length($y)
x=1,y=abcdefg,z=3,n=1

mlr put $n = length($nonesuch)
x=1,y=abcdefg,z=3,n=0

mlr put $n = length($*)
x=1,y=abcdefg,z=3,n=3

mlr put $n = length([])
x=1,y=abcdefg,z=3,n=0

mlr put $n = length([5,6,7])
x=1,y=abcdefg,z=3,n=3

mlr put $n = length({})
x=1,y=abcdefg,z=3,n=0

mlr put $n = length({"a":5,"b":6,"c":7})
x=1,y=abcdefg,z=3,n=3

